packages/react/src/file-upload/utils.test.tsx

```diff
@@ -53,3 +53,44 @@ describe('getErrorSummary', () => {
 		);
 	});
 });
+
+describe('formatFileSize', () => {
```

Comment:
These tests have been adapted from the tests in the `filesize` package.
https://github.com/avoidwork/filesize.js/blob/master/test/filesize_test.js